Sources/Widgets/Core/WidgetManager/Constants.d.ts

```diff
@@ -0,0 +1,25 @@
+export declare const ViewTypes = {
+  DEFAULT: 0,
+  GEOMETRY: 1,
+  SLICE: 2,
+  VOLUME: 3,
+  YZ_PLANE: 4,
+  XZ_PLANE: 5,
+  XY_PLANE: 6
+};
+
+export declare const RenderingTypes = {
+  PICKING_BUFFER: 0,
+  FRONT_BUFFER: 1
+};
+
+export declare const CaptureOn = {
+  MOUSE_MOVE: 0,
+  MOUSE_RELEASE: 1
+};
+
+export default {
```

Comment:
error TS2714: The expression of an export assignment must be an identifier or qualified name in an ambient context.

https://github.com/Microsoft/TypeScript/wiki/Breaking-Changes#arbitrary-expressions-are-forbidden-in-export-assignments-in-ambient-contexts